scan/src/contexts/GlobalContext.re

```diff
@@ -4,13 +4,21 @@ let context = React.createContext(ContextHelper.default);
 
 [@react.component]
 let make = (~children) => {
-  let financialOpt = PriceHook.get();
+  let (financialOpt, reload) = PriceHook.get();
 
   let data = {
     let%Opt financial = financialOpt;
     Some({financial: financial});
   };
 
+  React.useEffect1(
+    () => {
+      let intervalID = Js.Global.setInterval(() => {reload()}, 2000);
```

Comment:
Remove unnecessary `{`. Let's use `() => reload()`

Reply:
What's the type of `reload`? Can we just do ` Js.Global.setInterval(reload, ...`?